.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -106,4 +106,3 @@ stages:
         - template: ./templates/.builder-release-template.yaml
           parameters:
             artifactName: marinerv2-gen2
-    
```

Comment:
lets try to avoid spacing/newline changes to minimize the size of the change-set